### PR TITLE
gfxtablet: git-2013-10-21 -> 1.4

### PR DESCRIPTION
**Pull request by j1r1k**
###### Motivation for this change

Old git revision replaced with latest released version on github.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by grahamc** [comment]
LGTM I was able to compile them. The caveat though is I'm not actually able to test it, since ... well, I don't have a tablet.

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[pkgs/os-specific/linux/gfxtablet/default.nix, line 10 \[r1\]](https://reviewable.io:443/reviews/nixos/nixpkgs/17085#-KN-qdme0c5KGYXkgkBp:-KN-qdmfGBNTlm9hDwEb:-1485718697) ([raw file](https://github.com/nixos/nixpkgs/blob/50b690fd03dabb5c3e21c31e59ab2187d4b5f8a7/pkgs/os-specific/linux/gfxtablet/default.nix#L10)):*
> ```Nix
>     sha256 = "14ll9rls2mamllmpwjiv2dc6165plazv7ny9cliylafrwgb55l7p";
>     url = "https://github.com/rfc2822/GfxTablet.git";
>   };
> ```

What was generating this information? Should we have kept it around?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/nixos/nixpkgs/17085#-:-KN-qlB0oCZm1uEmtJDl:332723372)*

File: pkgs/os-specific/linux/gfxtablet/default.nix [hunk 1]
```diff
@@ -1,32 +1,28 @@
...
   ];
-in
-stdenv.mkDerivation {
-  inherit (s) name version;
-  inherit buildInputs;
-  src = fetchgit {
-    inherit (s) url sha256 rev;
+
+  src = fetchurl {
+    url = "https://github.com/rfc2822/GfxTablet/archive/android-app-${version}.tar.gz";
```
Comment (joachifm) [inline pkgs/os-specific/linux/gfxtablet/default.nix]: Please use `fetchFromGitHub` with their generated tarballs (the hash will change whenever github decides to change compression levels and whatnot). The hash can be pre-computed with `nix-prefetch-url --unpack`.

**Comment by j1r1k** [comment]
@joachifm: I have amended the commit to use `fetchFromGitHub`. Thanks for pointing out.

**Comment by j1r1k** [comment]
@grahamc: It can also be tested with android phone without stylus. Or we could potentially forge the network traffic generated by the app - I am not sure it is worth it tho. Let me know.

**Comment by j1r1k** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[pkgs/os-specific/linux/gfxtablet/default.nix, line 10 \[r1\]](https://reviewable.io:443/reviews/nixos/nixpkgs/17085#-KN-qdme0c5KGYXkgkBp:-KN3_Y4ehg2pzS82LdbM:412235638) ([raw file](https://github.com/nixos/nixpkgs/blob/50b690fd03dabb5c3e21c31e59ab2187d4b5f8a7/pkgs/os-specific/linux/gfxtablet/default.nix#L10)):*
<details><summary><i>Previously, grahamc (Graham Christensen) wrote…</i></summary>
> What was generating this information? Should we have kept it around?
</details>
Do you mean that I should have kept the structure of the file as it was?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/nixos/nixpkgs/17085)*

**Comment by grahamc** [comment]
I'm not that worried about actually testing it works, I'll trust it does.

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[pkgs/os-specific/linux/gfxtablet/default.nix, line 10 \[r1\]](https://reviewable.io:443/reviews/nixos/nixpkgs/17085#-KN-qdme0c5KGYXkgkBp:-KN3a86jvLfiv7v1BK9a:920682433) ([raw file](https://github.com/nixos/nixpkgs/blob/50b690fd03dabb5c3e21c31e59ab2187d4b5f8a7/pkgs/os-specific/linux/gfxtablet/default.nix#L10)):*
<details><summary><i>Previously, j1r1k (Jiri) wrote…</i></summary>
> Do you mean that I should have kept the structure of the file as it was?
</details>
I don't know either way, I just don't know why it was done like that.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/nixos/nixpkgs/17085#-:-KN3aGux3mElmWhClG8N:-2076638867)*

**Comment by grahamc** [comment]
:lgtm: @jo

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/nixos/nixpkgs/17085#-:-KN3at2IdLTCjuovhbEl:298895286)*

**Pull request merged**